docs/docs/09-rfcs/2023-03-08-wing-preview-environment.md

```diff
@@ -0,0 +1,282 @@
+---
+title: Wing Cloud Preview Environments
+description: This document describes the Wing Cloud Preview Environments feature.
+---
+
+# #3696 - Wing Cloud Preview Environments
+
+- **Author(s)**: @ainvoner, @skyrpex
+- **Submission Date**: 2023-08-03
+- **Stage**: Proposal
+- **Stage Date**: 2023-08-03
+
+This document describes the Wing Cloud Preview Environments feature.
+
+## Introduction
+
+The **Wing Cloud Preview Environments** project is designed to achieve two key objectives. Firstly, it aims to offer a preview environment solution for those interested in using winglang.
+Secondly, it seeks to enhance the winglang ecosystem and its toolchain by building a production application using the language.
+
+## Requirements
+
+Wing Cloud users can build and deploy Wing applications to an ephemeral (simulated) environment and use the Wing Console UI to interact with it.
+Automatically run tests on the preview environment once created and provide a UI for viewing the test results.
+Allow developers to collaborate with their teammates using a shared environment.
+Seamlessly integrating with GitHub.
+
+### Sign Up to Wing Cloud
+
+1. Goto https://wing.cloud and click on sign up.
+2. sign up to Wing Cloud using GitHub authentication.
+
+### Sign In to Wing Cloud
+
+1. Goto https://wing.cloud and click on sign in.
+2. If a developer was already signed in before, he will be automatically signed in to Wing Cloud.
```

Comment:
@ainvoner signing in and signing up are the same process. I'd simplify it to just "Sign In".